### ISSUE TITLE
Color doesn't get picked

### ISSUE THREAD

**Issue by end-4** [hyprwm/hyprpicker#40]
## The behaviour
- It shows the circular magnifier, but doesn't show the border with targeted color
- Clicking anyway just returns `#000000`
![image](https://github.com/hyprwm/hyprpicker/assets/97237370/f44cc796-d078-450e-8129-1c13a44b938d)

## Details
- Happens with the latest commit 3 days ago (03bc27be945a2c83f5b8089ae03030e10f56e371)
  - Before that it functions normally
- Hyprland version, if it matters:
```
Hyprland, built from branch main at commit 6e53c47e68c1d15032b755270568eb5485f2c684 dirty (renderer: fix missing premultiplication for border multisampling).
Tag: v0.28.0-10-g6e53c47e

flags: (if any)
```

**Comment by vaxerski** [comment]
oopsie woopsie that's a fucky-wucky indeed!

**Comment by vaxerski** [comment]
fixed in 5ba32686943f839d072426d9ffd172decaee0e3e
